src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2494,6 +2522,10 @@ export async function getHugoSymbolByEntrezGeneId(
     return gene.hugoGeneSymbol;
 }
 
+export function getHugoSymbolByChartTitle(title: string): string {
```

Comment:
is there a better way than parsing the title?

Reply:
this will be removed as well. I will get the hugogenesymbol using other data